### ISSUE TITLE
[TW-1450] Projects command should trigger running garbage collector

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1477]
_Tomas Babej on 2014-11-02T21:09:20Z says:_ 

 If you complete a task, and run the "task projects" commands, the task completion is not reflected in the task count of the project (shows one more task then it should).

This is apparently happening for performance reasons, but it might be not a bad idea to make projects command a exception to this rule.

Snippet from IRC logs:

(09:48:22 PM) bqf: tbabej: that would appear to be an issue with garbage collection. well, not an actual "issue", but a weirdness. garbage collection (moving tasks from pending.data to completed.data) only happens when a report displaying IDs is run. the projects report doesn't show IDs, so those completed tasks are still shown.
(09:50:21 PM) tbabej: is that happening for performance reasons?
(09:51:39 PM) bqf: tbabej: well... you'd have to ask pbeckingham. i think it might be a good idea to give the projects report special treatment and run GC despite not showing IDs.
(09:51:43 PM) pbeckingham: tbabej: GC is for performance reasons, yes.  We could always force project to run a GC.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-11-02T21:09:20Z
Modified: 2015-02-28T14:33:57Z
```

**Comment by taskwarrior** [comment]
_Tomas Babej on 2015-02-28T14:33:57Z says:_ 

 Fixed with: ffd6465661fee267dd8f6eb0bd9f81069e31101b
